superset/db_engine_specs/snowflake.py

```diff
@@ -74,3 +84,32 @@ def convert_dttm(cls, target_type: str, dttm: datetime) -> Optional[str]:
         if tt == "TIMESTAMP":
             return f"""TO_TIMESTAMP('{dttm.isoformat(timespec="microseconds")}')"""
         return None
+
+    @classmethod
+    @cache.memoize()
+    def get_function_names(
+        cls, database: "Database", schema: Optional[str]
+    ) -> List[str]:
+        """
+        Get a list of function names that are able to be called on the database.
+        Used for SQL Lab autocomplete. Returns global functions, excluding system
+        functions (starting with "SYSTEM$") and UDFs that are available in the chosen
+        schema.
+
+        :param database: The database to get functions for
+        :param schema: The schema to get functions for
+        :return: A list of function names useable in the database
+        """
+
+        def is_valid_function(func_row: pd.Series) -> bool:
```

Comment:
Nit. `is_valid_function` -> `_is_valid_function` since it's nested.

Reply:
Fixed.